astroquery/gemini/tests/test_remote.py

```diff
@@ -0,0 +1,45 @@
+""" Test Gemini Astroquery module.
+
+Remote tests to exercise interaction with the actual REST service.
+
+For information on how/why this test is built the way it is, see the astroquery
+documentation at:
+
+https://astroquery.readthedocs.io/en/latest/testing.html
+"""
+
+from astropy import units
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from astropy.tests.helper import remote_data
+
+from astroquery import gemini
+
+
+""" Coordinates to use for testing """
+coords = SkyCoord(210.80242917, 54.34875, unit="deg")
+
+
+@remote_data
+class TestGemini(object):
+    def test_observations_query_region(self):
+        """ test query against a region of the sky against actual archive """
+        result = gemini.Observations.query_region(coords, radius=0.3 * units.deg)
+        assert isinstance(result, Table)
```

Comment:
some check on the content of the Table would be nice in the tests, just to make sure data is returned rather than an empty table (which is the default behaviour for a nonsensical query, if I see correctly)

Reply:
I've added a length > 0 check.  That is still pretty naive but at least it will prevent the default causing a false pass.